app/src/router/module/explorer.js

```diff
@@ -6,11 +7,43 @@ const explorerRoutes = [
     meta: { requiresAuth: false }
   },
   {
-    path: 'chart/view/:uri',
-    name: 'ChartView',
-    component: () => import('@/pages/explorer/vega/view/VegaView.vue'),
+    path: 'visualization',
+    name: 'ExplorerVisualization',
     meta: { requiresAuth: false }
   },
+  {
+    path: 'create',
+    name: 'ExplorerCreate',
+    meta: { requiresAuth: false }
+  },
+  {
+    path: 'chart',
+    component: ChartBase,
+    meta: { requiresAuth: false },
```

Comment:
I'll suggest removing the meta tag from here since it's available for the children

Reply:
Fixed in [0b9d37a](https://github.com/Duke-MatSci/materialsmine/pull/111/commits/0b9d37a737dea71640b89c2cfb5f5d4c905d3a94)